dictionary.txt

```diff
@@ -926,3 +926,5 @@ _rp	₎
 \doublebarvee	⩢	
 \doublebarwedge	⩞	
 \perspcorrespond	⩞	
+\daleth  ℸ 
+
```

Comment:
Make sure to correct the spaces to tabs also
```suggestion
\daleth	ℸ	
```

Reply:
done!